app/views/users/my_certificate.html.erb

```diff
@@ -23,6 +23,7 @@
            <span style="font-size:20px"><%= t('certificate.line6') %></span> <br/><br/>
            <span style="font-size:25px"><i><%= t('certificate.line7') %></i></span><br>
            <span style="font-size:25px"><b><p id="date"></p></b></span><br>
+           <span style="font-size:15px"><b><a href="https://citsci.geog.mcgill.ca/"><%= t('certificate.line8') %></a></b></span><br>
```

Comment:
This will always go to the production site, never the dev site or staging site. You should use `<%= root_url %>` as the href value instead